test/ci/kokoro/config_generator.sh

```diff
@@ -14,22 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Create a config file for gsutil to use in Kokoro tests.
+# Create a .boto config file for gsutil to use in Kokoro tests.
 # https://cloud.google.com/storage/docs/gsutil/commands/config
 
 GSUTIL_KEY=$1
 API=$2
+OUTPUT_FILE=$3
 
-cat  << EOF
-[Credentials]
-gs_service_key_file = "$GSUTIL_KEY"
+cat > $3 <<- EOM
+	[Credentials]
```

Comment:
The `<<-` ignores tabs, right? I'd rather not have to tweak my editor settings to mess with this in the future (vim changes all tabs to spaces for me, for most file extensions). Could you put this back to left-aligned text?

Reply:
For sure! I'll undo the tabs and change `<<-` to `<<`.